c-dependencies/js-compute-runtime/builtins/backend.cpp

```diff
@@ -17,6 +19,617 @@
 #include "host_call.h"
 #include "js/Conversions.h"
 
+enum class Authentication {
```

Comment:
You can make all these `enum class` declarations subtype `uint8_t` to cut ~20 bytes out of the `Cipher` representation.
```suggestion
enum class Authentication : uint8_t {
```